rust/noosphere-ns/src/bin/orb-ns/main.rs

```diff
@@ -43,5 +55,39 @@ async fn main() -> Result<()> {
             }
             Ok(())
         }
+        CLICommand::Status { api_url } => {
+            let client = HTTPClient::new(api_url).await?;
+            let info = client.network_info().await?;
+            println!("{:#?}", info);
```

Comment:
Let's prefer `tracing::{info,debug,error,warn,trace}` to `println` throughout this change!